pkg/model/model.go

```diff
@@ -38,8 +41,60 @@ type Coeff struct {
 	InterceptDram float64 `csv:"intercept_dram"`
 }
 
+type EnergyPrediction struct {
+	Architecture   string
+	CPUTime        float64
+	CPUCycle       float64
+	CPUInstr       float64
+	ResidentMemory float64
+	CacheMisses    float64
+}
+
+type CategoricalFeature struct {
+	Name   string  `json:"name"`
+	Weight float64 `json:"weight"`
+}
+
+type NormalizedNumericalFeature struct {
+	Mean     float64 `json:"mean"`
+	Variance float64 `json:"variance"`
+	Weight   float64 `json:"weight"`
+}
+
+type CoreModelServerCoeff struct {
+	AllWeights struct {
+		BiasWeight           float64 `json:"Bias_Weight"`
```

Comment:
there are mixed upper and lower cases here, are they designed to be that way?

Reply:
I believe the naming convention is camel case but start upper case if public. Acronyms are also always capitalized. https://stackoverflow.com/questions/52358247/what-are-golang-struct-field-naming-conventions